boards/arm/stm32f411e_disco/doc/stm32f411e_disco.rst

```diff
@@ -0,0 +1,184 @@
+              LD7 (green) VBus and LD8 (red) over-current
+
+- Two pushbuttons (user and reset)
+- USB OTG with micro-AB connector
+- Extension header for LQFP100 I/Os for a quick connection to the prototyping board and an easy probing
+
+
+.. image:: img/en.stm32f411e-disco.jpg
+     :width: 533px
+     :align: center
+     :height: 500px
+     :alt: STM32F411E-DISCO
+
+More information about the board can be found at the `32F411EDISCOVERY website`_.
+
+Hardware
+********
+
+STM32F411E-DISCO Discovery kit provides the following hardware components:
+
+- STM32F411VET6 in LQFP100 package
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 100 MHz max CPU frequency
+- VDD from 1.7 V to 3.6 V
+- 512 KB Flash
+- 128 KB SRAM
+- GPIO with external interrupt capability
+- 1x12-bit, 2.4 MSPS ADC with 16 channels
+- DMA Controller
+- Up to 11 Timers (six 16-bit, two 32-bit, two watchdog timers and a SysTick timer)
+- USART/UART (3)
+- I2C (3)
+- SPI/I2S (5)
+- SDIO
+- USB 2.0 full-speed device/host/OTG controller with on-chip PHY
+- CRC calculation unit
+- 96-bit unique ID
+- RTC
+
+More information about STM32F411VE can be found here:
+       - `STM32F411VE website`_
+       - `STM32F411x reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32f411e_disco board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file :file:`boards/arm/stm32f411e_disco/stm32f411e_disco_defconfig`
+
+
+Pin Mapping
+===========
+
+STM32F411E-DISCO Discovery kit has 5 GPIO controllers. These controllers are responsible for pin muxing, input/output, pull-up, etc.
+
+For mode details please refer to `32F411EDISCOVERY board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+- LD3 : PD13
+- LD4 : PD12
+- LD5 : PD14
+- LD6 : PD15
+
+System Clock
+============
+
+STM32F411E-DISCO System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at 100MHz,
+driven internal oscillator.
```

Comment:
change to (I think):

     driven by the internal oscillator.

Reply:
OK, updated. Thank you!